DCCpp_Uno/src/PacketRegister.cpp

```diff
@@ -227,7 +227,7 @@ void RegisterList::readCV(const char *s) volatile{
 
     loadPacket(0,resetPacket,2,3);          // NMRA recommends starting with 3 reset packets
     loadPacket(0,bRead,3,5);                // NMRA recommends 5 verfy packets
-    loadPacket(0,resetPacket,2,1);          // forces code to wait until all repeats of bRead are completed (and decoder begins to respond)
+    loadPacket(0,bRead,3,1);          // forces code to wait until all repeats of bRead are completed (and decoder begins to respond)
```

Comment:
it would be better to send an idle packet at this point and move the reset packet load until *AFTER* the set of analog read calls. This would apply to all updates below.

Reply:
Is this good to go now?

Reply:
First, are we ready to merge this? and second, in the future aren't we going to ditch the development branch?

Reply:
I think this can be merged but we should create a tracking issue to clean this up per NMRA standards (send reset, send read/write, check acks, send reset only after acks).

As for where to merge, we can merge to master or development per the current plans. To update the target branch click edit on the PR and set the new branch accordingly.

Reply:
Totally agree, should merge but note we need to clean as Atani states.  I will try to research the NMRA documentation a bit further this week and can post the relevant sections so we can fix permanently.

Reply:
prog track ref: https://www.nmra.org/sites/default/files/s-9.2.3_2012_07.pdf